ports/ginkgo/portfile.cmake

```
@@ -42,6 +41,12 @@ vcpkg_cmake_install()
 vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/Ginkgo)
 vcpkg_fixup_pkgconfig()
 
+vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/share/ginkgo/GinkgoConfig.cmake" [[string(REPLACE "lib/cmake/Ginkgo" "" GINKGO_INSTALL_PREFIX "${GINKGO_CONFIG_FILE_PATH}")]] "")
+vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/share/ginkgo/GinkgoConfig.cmake" "GINKGO_INSTALL_PREFIX" "_IMPORT_PREFIX")
+vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/share/ginkgo/GinkgoConfig.cmake" "/lib/cmake/Ginkgo\"" "/share/ginkgo\"")
+vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/share/ginkgo/GinkgoConfig.cmake" "/lib/cmake/Ginkgo/Modules\"" "/share/ginkgo/Modules/Modules\"")
+vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/share/ginkgo/GinkgoConfig.cmake" "\"${SOURCE_PATH}/cmake/Modules/\"" "")
```

Should this be relative to _IMPORT_PREFIX?

---

Yes. It is currently relative to GINKGO_INSTALL_PREFIX, but GINKGO_INSTALL_PREFIX is wrong.

---

Right, but you replaced it with nothing, not a reference to _IMPORT_PREFIX

---

Ah you mean the last line. Yeah that line was wrong and it pointed to the same dir as the line above.

---

So the file is now
```cmake
set(GINKGO_INSTALL_MODULE_DIR "${_IMPORT_PREFIX}/share/ginkgo/Modules/Modules")
...
list(APPEND CMAKE_MODULE_PATH  "${GINKGO_INSTALL_MODULE_DIR}")
```
and was 
```cmake
set(GINKGO_INSTALL_MODULE_DIR "${GINKGO_INSTALL_PREFIX}/lib/cmake/Ginkgo/Modules")
...
list(APPEND CMAKE_MODULE_PATH "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/ginkgo/src/v1.4.0-262275ed0c.clean/cmake/Modules/" "${GINKGO_INSTALL_MODULE_DIR}")
```
and in `share/ginkgo/Modules/Modules` you find the files from `.../src/v1.4.0-262275ed0c.clean/cmake/Modules/`

---

```suggestion
vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/share/ginkgo/GinkgoConfig.cmake" "\"${SOURCE_PATH}/cmake/Modules/\"" "\\$\{_IMPORT_PREFIX\}")
```

To clarify, this is what I'm suggesting. I understand that it was wrong before (I tried this on my own box and looked at the output) but I think this would be a more in-keeping-with-what-the-authors intended fix?

---

Err, I wrote that wrong, I forgot to put `/cmake/Modules` back on, but you get the idea...

---

Ahh. But there is no `/cmake/Modules` folder in vcpkg_installed and I think there will never be one